### ISSUE TITLE
New version of go.opentelemetry.io/otel/sdk/metric ( v.035.0 ) breaks

### ISSUE THREAD

**Issue by derkan** [uptrace/opentelemetry-go-extra#91]
New version of [opentelemetry-go/metric](https://github.com/open-telemetry/opentelemetry-go/pull/3623) v.035.0, breaks this library.

```
# github.com/uptrace/opentelemetry-go-extra/otelsql
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:74:34: t.meter.SyncInt64 undefined (type metric.Meter has no field or method SyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:187:27: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:191:24: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:195:25: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:199:24: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:203:29: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:207:32: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:212:33: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:216:37: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:220:37: meter.AsyncInt64 undefined (type metric.Meter has no field or method AsyncInt64)
../../../../go/pkg/mod/github.com/uptrace/opentelemetry-go-extra/otelsql@v0.1.18/otel.go:220:37: too many errors
....
../../../../go/pkg/mod/github.com/uptrace/uptrace-go@v1.11.8/uptrace/metrics.go:69:14: undefined: metric.InstrumentKindSyncCounter
../../../../go/pkg/mod/github.com/uptrace/uptrace-go@v1.11.8/uptrace/metrics.go:70:10: undefined: metric.InstrumentKindAsyncCounter
../../../../go/pkg/mod/github.com/uptrace/uptrace-go@v1.11.8/uptrace/metrics.go:71:10: undefined: metric.InstrumentKindSyncHistogram

```

**Comment by vmihailenco** [comment]
Should be fixed by v0.1.19

**Comment by derkan** [comment]
thank you for response 🥇
